pkgs/top-level/ocaml-packages.nix

```diff
@@ -1143,6 +1143,8 @@ let
 
     reason = callPackage ../development/compilers/reason { };
 
+    reason-native = callPackage ../development/ocaml-modules/reason-native { };
```

Comment:
```suggestion
    reason-native = lib.recurseIntoAttrs (callPackage ../development/ocaml-modules/reason-native { });
```

Reply:
What does it solve? It was working before this (at least it seemed).

Reply:
What was “working before”? `nix search`?
https://nixos.org/manual/nixpkgs/stable/#function-library-lib.attrsets.recurseIntoAttrs